be/src/storage/rowset/segment_writer.cpp

```diff
@@ -146,6 +146,10 @@ Status SegmentWriter::init(const std::vector<uint32_t>& column_indexes, bool has
         // TODO(mofei) refactor it to type specification
         opts.need_zone_map = column.is_key() ||
                              (_tablet_schema->keys_type() == KeysType::DUP_KEYS && is_zone_map_key_type(column.type()));
```

Comment:
Can we use zonemap in the `PRIMARY_KEYS `  key type by default for supported column types like `DUP_KEYS `?

Reply:
Yes, but this need discuss. Keep the default behavior first.